pkg/config/load.go

```diff
@@ -90,12 +101,23 @@ func InfoFromPath(configFilePath string) (*Info, error) {
 		branch = branch[:i]
 	}
 
+	pConfig := Prowgen{}
+	b, err := ioutil.ReadFile(filepath.Join(configSpecDir, prowgenConfigFile))
+	if err != nil && !os.IsNotExist(err) {
+		return nil, fmt.Errorf("prowgen config found but couldn't read the file: %v", err)
+	}
+
+	if err := yaml.Unmarshal(b, &pConfig); err != nil {
```

Comment:
I think we should be more explicit here, and not even attempt to `Unmarshal` when `ReadFile` failed. It is true that it will probably work, because `b` is probably empty (but is not documented as such), and `yaml.Unmarshal` works on empty string, but I do not like relying on undocumented behavior.

Reply:
addressed